docs/generated/packages/react/generators/host.json

```diff
@@ -84,7 +84,7 @@
         "x-priority": "internal"
       },
       "skipNxJson": {
-        "description": "Skip updating workspace.json with default options based on values provided to this app (e.g. babel, style).",
+        "description": "skip updating nx.json with default options based on values provided to this app (e.g. babel, style).",
```

Comment:
Are we intentionally lower casing the first word here? 
Other options in the schema are title cased for the first word.